android/src/main/java/com/tailscale/ipn/DnsConfig.java

```diff
@@ -231,10 +231,10 @@ String getDnsServersFromSystemProperties() {
 
 	String intToInetString(int hostAddress) {
 		return String.format(java.util.Locale.ROOT, "%d.%d.%d.%d",
-			(byte)(0xff & hostAddress),
-			(byte)(0xff & (hostAddress >> 8)),
-			(byte)(0xff & (hostAddress >> 16)),
-			(byte)(0xff & (hostAddress >> 24)));
+			(0xff & hostAddress),
+			(0xff & (hostAddress >> 8)),
+			(0xff & (hostAddress >> 16)),
+			(0xff & (hostAddress >> 24)));
 	}
```

Comment:
Perfect little thing to add a test for if we want to get fancy and start learning how to do Java testing 😉

Reply:
Added a test.